src/styles.ts

```diff
@@ -11,7 +11,7 @@ export function getAll() {
   // Convert rules array to a string
   return styleSheet
     .rules()
-    .reduce((combinedRules: string, rule: {cssText: string; }) => combinedRules + rule.cssText, '')
+    .reduce((combinedRules: string, rule: { cssText: string }) => combinedRules + rule.cssText, '')
```

Comment:
I think you may be able to remove the type here and it will be inferred correctly.